src/go/k8s/apis/redpanda/v1alpha1/console_types.go

```diff
@@ -42,6 +38,25 @@ type ConsoleSpec struct {
 	Version string `json:"version"`
 }
 
+// Config for a HTTP server
+// Grabbed from https://github.com/cloudhut/common
+// Added JSON tags. We can have this merged from upstream.
+type RESTConfig struct {
+	ServerGracefulShutdownTimeout time.Duration `yaml:"gracefulShutdownTimeout" json:"gracefulShutdownTimeout"`
+
+	HTTPListenAddress      string        `yaml:"listenAddress" json:"listenAddress"`
```

Comment:
I've not followed the whole conversation. Are you going with representing the full console config as CRD?

I think the approach is good, especially until the api is alpha or console does not make backward incompatible changes.

Reply:
I think Rafal went on implementing the whole Console config as CRD for flexibility in configuration. We may or may not need all of them though.